src/StackAdmin/Azs.AzureBridge.Admin/Tests/src/Common.ps1

```diff
@@ -1,23 +1,27 @@
 ﻿
-$global:Location = "local"
-$global:TenantVMName = "502828aa-de3a-4ba9-a66c-5ae6d49589d7"
+# Test Variables
+$global:SkippedTests = @(
+    "TestDownloadAzsAzureBridgeProductPipeline",
```

Comment:
@deathly809 why are we skipping these tests?

Reply:
There was a reason, @viananth would have more information as she worked on these tests.

Reply:
The test is not a valid one and should be removed. Downloading the product requires the activation name and it is not really pipeable from Get-AzsAzureBridgeProduct

Reply:
@bganapa @deathly809 discussed with Bala offline about the skipped tests -- if we are skipping a test that your team (or a service team) doesn't plan on trying to re-enable in the future, then we should remove the test. If it's the case where it's possible that the test will be re-enabled in the future, then we should leave a comment above the skipped test (in the `Common.ps1` file) briefly explaining why the test is being skipped